lib/policy.py

```diff
@@ -2151,7 +2151,7 @@ def ParseFile(filename, definitions=None, optimize=True, base_dir='',
 
 
 def ParsePolicy(data, definitions=None, optimize=True, base_dir='',
-                shade_check=False):
+                shade_check=False, filename=''):
```

Comment:
I see you are adding this to ParsePolicy but should you did not add this to the call from ParseFile. Would it not be a good idea to add it there?